lib/schemapack.js

```diff
@@ -252,113 +255,119 @@ class SchemaPack {
       return callback(new OpenApiErr('The schema must be validated before attempting conversion'));
     }
 
-    // this cannot be attempted before validation
-    specComponentsAndUtils = { concreteUtils };
-    Object.assign(specComponentsAndUtils, concreteUtils.getRequiredData(this.openapi));
-
-    // create and sanitize basic spec
-    openapi = this.openapi;
-    openapi.servers = _.isEmpty(openapi.servers) ? [{ url: '/' }] : openapi.servers;
-    openapi.securityDefs = _.get(openapi, 'components.securitySchemes', {});
-    openapi.baseUrl = _.get(openapi, 'servers.0.url', '{{baseURL}}');
-
-    // TODO: Multiple server variables need to be saved as environments
-    openapi.baseUrlVariables = _.get(openapi, 'servers.0.variables');
-
-    // Fix {scheme} and {path} vars in the URL to :scheme and :path
-    openapi.baseUrl = schemaUtils.fixPathVariablesInUrl(openapi.baseUrl);
-
-    // Creating a new instance of a Postman collection
-    // All generated folders and requests will go inside this
-    generatedStore.collection = new sdk.Collection({
-      info: {
-        name: _.isEmpty(_.get(openapi, 'info.title')) ? COLLECTION_NAME : _.get(openapi, 'info.title')
+    convertSwaggerToOpenapi(concreteUtils, this.openapi, (error, newOpenapi) => {
```

Comment:
Can we rename this function to `convertToOAS30IfSwagger` and also add an additional comment above around that we only convert if swagger is found otherwise `this.openapi` remains same.